pkg/proxy/mapping.go

```diff
@@ -19,11 +19,12 @@ package proxy
 import (
 	"context"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"net/http/httputil"
 	"net/url"
+	"os"
 
+	"k8s.io/component-base/metrics/legacyregistry"
```

Comment:
Doesn't have to be solved in this PR if this is good enough to get things going forward, but should we be using metrics/registry instead of metrics/legacyregistry?

Reply:
Let's file an issue to figure out what to do. Being consistent for now is 👍